drivers/adc/adc_stm32.c

```diff
@@ -232,6 +240,12 @@ static const uint32_t table_samp_time[] = {
 /* External channels (maximum). */
 #define STM32_CHANNEL_COUNT		20
 
+/*
+ * Number of ADC clock cycles to wait before starting calibration and after
+ * enabling ADC for STM32F1 and STM32F37x
+ */
+#define STM32F1_ADC_DELAY_CALIB_ENABLE_ADC_CYCLES	2
```

Comment:
this is: 
#define STM32F1_ADC_DELAY_CALIB_ENABLE_ADC_CYCLES  LL_ADC_DELAY_ENABLE_CALIB_ADC_CYCLES

Reply:
Correct, I didn't see that. There is yet another macro for STM32F37x. I'll update this part